examples/typescript-server.ts

```diff
@@ -16,7 +16,7 @@ import * as cors from 'cors'
 import { createReadStream } from 'fs'
 import * as http from 'http'
 
-const server = fastify()
+const server = fastify<http.Server, http.IncomingMessage, http.ServerResponse>()
```

Comment:
this should not be needed.

Reply:
But `server.server` is missing type without those generics.